mmv1/third_party/terraform/tests/data_source_google_cloud_asset_resources_search_all_test.go.erb

```diff
@@ -28,8 +28,6 @@ func TestAccDataSourceGoogleCloudAssetResourcesSearchAll_basic(t *testing.T) {
 						"results.0.display_name", regexp.MustCompile(project)),
 					resource.TestMatchResourceAttr("data.google_cloud_asset_resources_search_all.resources",
 						"results.0.name", regexp.MustCompile(fmt.Sprintf("//cloudresourcemanager.googleapis.com/projects/%s", project))),
-					resource.TestMatchResourceAttr("data.google_cloud_asset_resources_search_all.resources",
```

Comment:
Do we know why this check is no longer needed?

Reply:
See description of the PR- they defined the field appearing as not guaranteed, and it no longer appears